lib/citizens_advice_form_builder/form_builder.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require "action_view"
+
+module CitizensAdviceFormBuilder
+  class FormBuilder < ActionView::Helpers::FormBuilder
+    def cads_text_field(attribute, label: nil)
+      label_text = label
+
+      @template.content_tag(:fieldset) do
```

Comment:
I think we shouldn't have the fieldset around the label and text field. Disclaimer: I'm assuming this will be different when the design system components are added and this is for example purposes, haven't looked at the other PR yet, so apologies if that's the case.